src/Core/Tools/SendFeatures/Commands/Interfaces/IAnonymousSendCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Bit.Core.Tools.Entities;
+
+namespace Bit.Core.Tools.SendFeatures.Commands.Interfaces;
+
+/// <summary>
+/// AnonymousSendCommand interface provides methods for managing anonymous Sends.
+/// </summary>
+public interface IAnonymousSendCommand
+{
+    /// <summary>
+    /// Gets the Send file download URL for a Send object.
+    /// </summary>
+    /// <param name="send">Send object to help get file download url and validate file</param>
+    /// <param name="fileId">FileId get file download url</param>
+    /// <param name="password">Password will be validated and used to determine access</param>
```

Comment:
⛏️ You're describing the expected behavior and usage of these parameters. Present tense is more clear. 

🎨 You should also communicate any properties of the parameter that are required but not captured in its type.

```suggestion
    /// <param name="password">A hashed and base64-encoded password. This is compared with the send's password to authorize access.</param>
```

Reply:
Add verbiage